app/views/posts/show.html.erb

```diff
@@ -0,0 +1,11 @@
+<h1>Showing selected post</h1>
+
+<div class="jumbotron">
+  <% if signed_in? %> 
```

Comment:
* [ ] There shoud be absolutely NO logic in views.

  > Views are only for displaying information that became available to it from its corresponding controller not generating it. **Resources**
* [Where Do I Put My Code?](http://codefol.io/posts/Where-Do-I-Put-My-Code)
* [The Beginner's Guide to Rails Helpers](https://mixandgo.com/learn/the-beginners-guide-to-rails-helpers)
* [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/section/no_logic_in_helpers_requirement.png) from Pathwright